recipes/bob.core/build.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+
+if [[ `uname` == 'Darwin' ]]; then
+	MACOSX_VERSION_MIN=10.7
```

Comment:
Try setting `CMAKE_OSX_DEPLOYMENT_TARGET=MACOSX_VERSION_MIN`. Also, try adding `export` in front of all the `bash` variables and let's see how that goes.